app/index.js

```diff
@@ -15,7 +15,7 @@ var marked        = require('marked');
 var validator     = require('validator');
 var extend        = require('extend');
 var hogan         = require('hogan-express');
-var basic_auth    = require('basic-auth-connect');
+var session       = require('express-session');
```

Comment:
@gilbitron - Do you feel we should keep HTTP Basic as an alternative option and then default to Matthias' session auth? Or just remote HTTP Basic entirely?